pkg/controller/dbaasregistration/configmap.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2021 MongoDB.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dbaasregistration
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	name      = "mongodb-atlas-registration"
```

Comment:
if you are following YAML based approach, is that possible to read the  YAML file from a directory, instead of putting it into constant? WDYT?

Reply:
Either way is fine, however putting it to a local file is more complicated, as we would have to change the docker file to have the file included. I just chose a simpler approach.